corehq/apps/reminders/templates/reminders/partial/scheduled_reminders.html

```diff
@@ -11,9 +11,9 @@
                         nickname: "{{ data.handler_name }}",
                         date: "{{ data.next_fire.date }}",
                         time: "{{ data.next_fire.time }}",
-                        case_name: "{{ data.case_name }}",
+                        case_name: {{ data.case_name|JSON }},
```

Comment:
nit but could potentially be easier to understand the intention if you kept in quotes and used `|escapejs` instead (i think that would accomplish the same thing)